server/public/styles.css

```diff
@@ -0,0 +1,78 @@
+*{
+  box-sizing:border-box;
+}
+
+.headerBackground{
+  background-color: #2c2c2c;
+}
+
+header{
+  padding: 15px 25px;
+}
+
+header .title{
+  color: #ffffff;
+  font-size: 25px;
+  font-weight:250;
+}
+
+header .logo{
+  margin:0px 5px;
+  transition:all ease .15s;
+  font-weight:600;
+}
+
+header .logo:hover{
+  color:#91EE91;
+}
+
+.productImage{
+  width:100%;
+  height:250px;
+  object-fit:scale-down;
+  margin-top:25px;
+}
+
+body{
+  background-color:#f8f9fa;
+}
+
+.productImageDiv{
+  width:100%;
+}
+
+.card{
+  height:100%;
+  box-shadow: 0px 0px 7px #d5d5d5;
+  border: none;
+  transition:all ease .1s;
+}
+.card:hover{
+  height:100%;
+  box-shadow: 0px 5px 10px #c5c5c5;
+  border: none;
+  transform:translate(0px,-7px);
+}
+
+.productDescription{
+  padding:25px 10px 10px;
+  margin:5px;
+}
+
+.cardContainer{
+  margin-top:25px;
+}
```

Comment:
Bootstrap already has spacing and color helpers. Can you use any of those and eliminate some custom css?